python/humbug/exceptions.py

```diff
@@ -0,0 +1,16 @@
+"""
+Humbug exceptions for inheritance by created user classes.
+"""
+from .report import Reporter
+
+
+class ExceptionWithReporting(Exception):
+    def __init__(self, bugout_reporter: Reporter, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        bugout_reporter.error_report(self, publish=True)
+
+
+class ValueErrorWithReporting(ValueError):
```

Comment:
I don't think we should create a different exception class for each of the Python native exception classes. It's best to just define `ExceptionWithReporting` and let users inherit from it as they want.

In my experience, most people just define their own custom exception hierarchies starting from `Exception` in the first place.

Reply:
Saved only ExceptionWithReporting